Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Golang build container
-FROM golang:1.12.10-alpine
+FROM golang:1.13.1-alpine
```

Comment:
Let's remove this change for now and fix that in separate PR together with CircleCI/build script update.

Reply:
OK, will do.